README.md

```diff
@@ -20,8 +20,9 @@
 
 ### What is Modin?
 
-Modin is a drop-in replacement for [pandas](https://github.com/pandas-dev/pandas). While pandas is
-single-threaded, Modin lets you instantly speed up your workflows by scaling pandas so it uses all of your
+Modin is a drop-in replacement for [pandas](https://github.com/pandas-dev/pandas) with 
```

Comment:
Maybe we should change this paragraph a bit? I wouldn't like to change/supplement the statement `Modin is a drop-in replacement for pandas`. A suggestion:

```
Modin is a drop-in replacement for pandas. While pandas is single-threaded,
Modin lets you instantly speed up your workflows by scaling pandas so it uses all of your
cores. Modin works especially well on larger datasets, where pandas becomes painfully slow or runs
out of memory. Also, Modin comes with the additional APIs to improve user experience.
```